dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/quartz/QuartzExecutors.java

```diff
@@ -150,7 +149,6 @@ public static QuartzExecutors getInstance() {
      */
     private void init() {
         try {
-            PropertyUtils.loadPropertyFile(DATASOURCE_PROPERTIES);
```

Comment:
I think this line just want to get variable `SPRING_DATASOURCE_DRIVER_CLASS_NAME ` in datasource.properties . and in this case we just have `SPRING_DATASOURCE_DRIVER_CLASS_NAME` in our env, Am I right @lenboo @CalvinKirs @kezhenxu94

Reply:
This is deleted on dev already, feel free to delete it :)

Reply:
@kezhenxu94 Do you have time to see other changes?